CHANGELOG.md

```diff
@@ -1,3 +1,20 @@
+### 0.9.0
+
+- Pin to Pagy 9.x.
```

Comment:
I would just say:

```
- BREAKING: Use Pagy 9.x. This contains breaking changes, see [CHANGELOG.md](https://github.com/ddnexus/pagy/blob/master/CHANGELOG.md#version-900)
- BREAKING: Rename `items` parameter to `limit` to align with latest Pagy naming convention.
```

Remove the rest, it's too verbose and doesn't tell much to the actual user :)